docs/docs/channels/email/emailjs.md

```diff
@@ -1,23 +1,27 @@
-# EmailJS
+# Email.js
 
-[EmailJS](https://www.emailjs.com/) helps to send emails using client-side technologies only. No server is required – just connect EmailJS to one of the supported email services, create an email template, and use our Javascript library to trigger an email. You can use EmailJS to send the content of a form filled by your users directly to your email address, using a simple connection provided by EmailJS.
+You can use the [Email.js](https://www.emailjs.com/) provider to send transactional emails to your customers using the Novu Platform with a single API to create multi-channel experiences.
 
 ## Getting Started
 
-You have to simply create an account on EmailJS and verify your email address after that you can use API or CDN provided by EmailJS to send a response of form to your email address.
+To use the Email.js channel, you will need to create a Email.js account and add your API key to the Email.js integration on the Novu platform.
 
-## How to connect a form to your email?
+## Find the API Key
 
-- Create a simple HTML form.
-- Create an account on EmailJS and verify your email.
-- Create a new service and do all the settings.
-- Create a connection using CDN or API.
+To find your Email.js API key, log into your Email.js account and navigate to the [API Keys](https://dashboard.emailjs.com/admin/account) page.
+It is suggested that you create a new API key for use with Novu. To successfully send emails, you will need to add the following permissions to your API key:
```

Comment:
This bit doesn't make sense. It is incomplete.
Also I couldn't find any recommendation from Email.js to create a different API key for different consumers from the same owner.